### PR TITLE
Add Non-Vanilla Dye Support to ZTones Blocks within Assembler

### PR DESCRIPTION
**Pull request by QuantumBlink1337**
In response to: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13921

image
image

I rewrote the Ztones recipes to better allow for additional ingredients to be usable in assembling. 

This is my first PR for GregTech: New Horizons - my friends and I encountered this issue in our own playthrough, so I gave it a go at fixing.

### CODE REVIEW

File: src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import static gregtech.api.util.GT_RecipeBuilder.SECONDS;
 import static gregtech.api.util.GT_RecipeBuilder.TICKS;
 
-import java.util.List;
+import java.util.*;
```
Comment (Caedis) [inline src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:13]: Avoid any wildcard imports

Comment (QuantumBlink1337) [reply src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java]: Latest commit fixes this, thank you

**Comment by Dream-Master** [comment]
@QuantumBlink1337 still star imports in your pr.

**Review by Ethryan — APPROVED** [review]
Just waiting for the check to finish, but on a quick look it looks fine.

**Pull request merged**